src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -53,8 +53,8 @@ internal sealed class VisualStudioProject
         private string _filePath;
         private CompilationOptions _compilationOptions;
         private ParseOptions _parseOptions;
+        private CompilationOutputs _compilationOutputs;
```

Comment:
is it just to be on the safe sid that this is the base type, and not the derived type?  Or is this very intentional because you'll have a different impl sometimes?  For example does EnC have it's own project type that would do things differently?

Reply:
It's intentional. I assume we can have projects that compile in-memory (e.g. ASP.NET).